gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/MapContainerViewModel.java

```diff
@@ -118,9 +137,19 @@
   /** The currently selected feature on the map. */
   private final BehaviorProcessor<Optional<Feature>> selectedFeature =
       BehaviorProcessor.createDefault(Optional.empty());
+
+  /** The currently selected layer for the polygon drawing. */
+  private final BehaviorProcessor<Optional<Layer>> selectedLayer =
+      BehaviorProcessor.createDefault(Optional.empty());
+
+  private final BehaviorProcessor<Optional<Project>> selectedProject =
+      BehaviorProcessor.createDefault(Optional.empty());
+
   /* UI Clicks */
   @Hot private final Subject<Nil> selectMapTypeClicks = PublishSubject.create();
   @Hot private final Subject<Point> addFeatureButtonClicks = PublishSubject.create();
+  @Hot private final Subject<Point> confirmButtonClicks = PublishSubject.create();
+  @Hot private final Subject<Nil> cancelButtonClicks = PublishSubject.create();
```

Comment:
This should be removed. Possible merge conflict.

Reply:
removed.